codecov_auth/commands/owner/interactors/get_uploads_number_per_user.py

```diff
@@ -18,11 +16,23 @@ def execute(self, owner: Owner):
         if monthly_limit is not None:
             # This should be mirroring upload/helpers.py::check_commit_upload_constraints behavior
             # We should put this in a centralized place
+            created_at = datetime.utcnow() - timedelta(days=30)
+            if (
+                plan_service.trial_status == TrialStatus.EXPIRED.value
+                and plan_service.has_trial_dates
+            ):
+                trial_duration_in_days = plan_service.trial_duration_in_days()
+                time_since_trial_end_in_days = (
+                    datetime.utcnow() - plan_service.trial_end_date
+                ).days
+                if time_since_trial_end_in_days < trial_duration_in_days:
+                    created_at = plan_service.trial_end_date
+
```

Comment:
Isn't there the case where the trial start and end dates are within the last month? For example,
Today (now): Apr 30
Trial End: Apr 25
Trial Start: Apr 11
-> This will compute the count of interval from Apr 25 to Apr 30 (now), but it is missing Apr 1 to Apr 11 interval

Reply:
hmmm, you're right. The solution is to exclude uploads that happened during the trial. I think the solution is to count the amount of days someone was on trial, add those to the time delta, so `created_at__gte= now - (30 + time_on_trial)`, and then exclude the uploads during the days of trial